File: Prob03.java
import java.io.*;

public class Prob03 {

    private static final String INPUT_FILE_NAME = "Prob03.in.txt";

    public static void main(String[] args) {
        try {
            File f = new File(INPUT_FILE_NAME);
            FileReader fr = new FileReader(f);
            BufferedReader br = new BufferedReader(fr);

            int n = Integer.parseInt(br.readLine());

            String currLine = null;
            for(int i = 0; i < n; i++) {
                currLine = br.readLine();
                String[] parts = currLine.split(" ");
                int num1 = Integer.parseInt(parts[0]);
                int num2 = Integer.parseInt(parts[1]);
                System.out.print((num1 + num2) + " ");
                System.out.println(num1 * num2);
            }

            br.close();
            fr.close();
        } catch(Exception e) {
            e.printStackTrace();
        }
    }
}

File: Prob03/Prob03.py
def main():
    with open("Prob03.in.txt") as f:
        myList = [line.rstrip('\n') for line in f]
        n = int(myList[0])
        for i in range(1, n + 1):
            parts = myList[i].split()
            num1 = int(parts[0])
            num2 = int(parts[1])
            print((num1 + num2), end = "")
            print(" ", end = "")
            print(num1 * num2)


if __name__ == "__main__": main()
